app/schemas/schemas.coffee

```diff
@@ -90,6 +99,16 @@ me.extendNamedProperties = (schema) ->
   schema.properties = {} unless schema.properties?
   _.extend(schema.properties, namedProps())
 
+
+# Named props for new schemas using json draft 07 
+namedPropsNew = ->
```

Comment:
Should this name refer to the json draft version?

Reply:
Probably, couldnt think of a better name. How about `namedPropsNewJson` or `namedPropsJson07` ?

Reply:
@AndrewJakubowicz merging the PR but keeping this question open. Will address this in the next PR based on what we decide here.

Reply:
`namedPropsJson07` is definitely specific and clear. I am also be happy with `namedPropsNew`.
Either decision is great.